auth-manifest/types/src/lib.rs

```diff
@@ -0,0 +1,168 @@
+/*++
+
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+   lib.rs
+
+Abstract:
+
+    File contains data structures for the image authorization manifest bundle.
+
+--*/
+
+#![no_std]
+
+use core::ops::Range;
+
+use caliptra_image_types::*;
+use core::default::Default;
+use memoffset::span_of;
+use zerocopy::{AsBytes, FromBytes};
+use zeroize::Zeroize;
+
+pub const AUTH_MANIFEST_MARKER: u32 = 0x4154_4D4E;
+pub const AUTH_MANIFEST_IMAGE_METADATA_MAX_COUNT: usize = 16;
+pub const AUTH_MANIFEST_VENDOR_SIGNATURE_REQURIED_FLAG: u32 = 0x1;
+
+#[repr(C)]
+#[derive(AsBytes, FromBytes, Default, Debug, Clone, Copy, Zeroize)]
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
+pub struct AuthManifestPubKeys {
+    pub ecc_pub_key: ImageEccPubKey,
+    #[zeroize(skip)]
+    pub lms_pub_key: ImageLmsPublicKey,
+}
+
+#[repr(C)]
+#[derive(AsBytes, FromBytes, Default, Debug, Clone, Copy, Zeroize)]
+pub struct AuthManifestPrivKeys {
+    pub ecc_priv_key: ImageEccPrivKey,
+    #[zeroize(skip)]
```

Comment:
Why are we skipping zeroizing the LMS private key?

Reply:
This is a good question, it looks like it's the case for all LMS keys and signatures here.

Reply:
This is being done for the FW image as well. I don't have the context for why that was done originally. We'll review this as a separate task.